index.html

```diff
@@ -270,7 +270,8 @@ <h2 class="contact-heading">Get started</h2>
                 ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
                 quis nostrud exercitation ullamco laboris nisi ut aliquip ex 
                 ea commodo consequat. </p>
-            <form class="form" action="https://formspree.io/f/mjvlwyao" method="POST">
+            <small></small>
+            <form class="form" id="form" action="https://formspree.io/f/mjvlwyao" method="POST">
```

Comment:
- [ ] Kindly make the error message near the submit button as required in the requirements.

![image](https://user-images.githubusercontent.com/35535130/156486288-f9026892-efae-4016-9243-35a845e5c3ae.png)